resources/helm/development.yaml

```diff
@@ -2338,6 +2356,9 @@ mod-quick-marc:
 mod-reading-room:
   extraEnvVars: []
   integrations:
+    okapi:
```

Comment:
mod-reading-room don't need this integration

Reply:
Done